iwyu_tool.py

```diff
@@ -86,10 +86,21 @@ def clang_formatter(output):
     'clang': clang_formatter
 }
 
+
 def get_output(cwd, command):
     """ Run the given command and return its output as a string. """
+    # https://stackoverflow.com/a/4453495
+    # do not alter the current environment,
+    # but copy all of it for the child process,
+    # then prepend the directory of iwyu_tool.py to PATH
+    env = os.environ.copy()
+    try:
+        env["PATH"] = os.path.dirname(__file__) + os.pathsep + env["PATH"]
```

Comment:
Prevalent style in this file is single-quoted strings. I found one example of a double-quoted string, but that was probably just overlooked.